rfcs/20210510-binary-size-reduction/ISA_extension.md

```diff
@@ -0,0 +1,117 @@
+# Build knobs controlling binary size extension.
+
+## Problem Statement.
+This follows up this
+[RFC](https://github.com/oneapi-src/oneDNN/tree/rfcs/rfcs/20210510-binary-size-reduction)
+and proposes additional build knob to control functionality by ISA. It may
+provide additional save up to 5 MB, which is around ten percent (~10%) from
+current full binary size, based on selected ISA.
```

Comment:
Do you plan to this new build knob to be CPU? The GPU runtime can probably also benefit from disabling implementations for certain architectures. Does it make sense to generalize the build knob so that we can enable it for both CPU and GPU runtimes?

Reply:
For GPU it sounds like it better be a different knob in case one wants to mix-and-match them together. Better to add `CPU` in name then.

Reply:
> Better to add CPU in name then.

Sounds good to me.

Reply:
Closing this item for sake of RFC promotion. GPU add-on should be considered separately. Thanks.